src/controller.coffee

```diff
@@ -214,6 +214,8 @@ exports.Editor = class Editor
     # ## Tracker Events
     # We allow binding to the tracker element.
     dispatchMouseEvent = (event) =>
+      # Firefox doens't have a global event object
+      window.event = event
```

Comment:
ahh right, i had just typed `window.event` and it was undefined, but that's of course because I wasn't in the process of dealing with an event. Interesting the mozilla article from above says it's IE only.

Modifying the global namespace as a workaround feels a little bit scary to me. I worry that by committing this fix, we make finding the underlying issue (dependence on the global) something we're less likely to actually do, and that we could be introducing other subtle differences in behavior (tho i'm not sure that's too big a worry in this case).

I'd feel better if we tracked down the real bug, and maybe if that proves particularly difficult commit this workaround. What do you think?